src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/04-create-cluster-create-services.e2e-spec.ts

```diff
@@ -0,0 +1,36 @@
+import { CreateClusterWizardHelper } from 'cypress/integration/cluster/create-cluster.po';
+
+describe('Create cluster create services page', () => {
+  const createCluster = new CreateClusterWizardHelper();
+
+  beforeEach(() => {
+    cy.login();
+    Cypress.Cookies.preserveOnce('token');
+    createCluster.navigateTo();
+    createCluster.createCluster();
+    cy.get('button[aria-label="Next"]').click();
+    cy.get('button[aria-label="Next"]').click();
+  });
+
+  it('should check if nav-link and title contains Create Services', () => {
+    cy.get('.nav-link').should('contain.text', 'Create Services');
+
+    cy.get('.title').should('contain.text', 'Create Services');
+  });
+
+  describe('when Orchestrator is available', () => {
+    it('should create an rgw service', () => {
+      createCluster.addService('rgw');
+
+      createCluster.checkServiceExist('rgw.rgw', true);
+    });
+
+    it('should create and delete an ingress service', () => {
+      createCluster.addService('ingress');
+
+      createCluster.checkServiceExist('ingress.rgw.rgw', true);
+
+      createCluster.deleteService('ingress.rgw.rgw', 60000);
```

Comment:
The wait time can be removed
```suggestion
      createCluster.deleteService('ingress.rgw.rgw');